docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contribution to Tartiflette
+
+:tada: Thanks for being here :tada:
+
+* [Code of Conduct](#code-of-conduct)
+* [How can I contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+* [Changelog](#changelog)
+
+## Code of conduct
+
+We expect from each contributor a respect for the [Code of Conduct](./CODE-OF-CONDUCT.md) which governs the **Tartiflette** project.
+
+## How can I contribute?
+### Reporting bugs
+
+To report a bug, you have to [create an issue on github](https://github.com/dailymotion/tartiflette/issues/new) and follow the following guidelines.
```

Comment:
To report a bug, please [create an issue on github](https://github.com/dailymotion/tartiflette/issues/new) and follow the guidelines below.